ctd/utilities.py

```diff
@@ -87,7 +85,13 @@ def read_file(fname, compression=None):
         name = zfile.namelist()[0]
         cfile = zfile.open(name)
     else:
-        cfile = open(fname, 'rb')
-    text = cfile.read().decode(encoding='utf-8', errors='replace')
+        # opening without 'rb' and reading without decode because you loose
+        #    the special charcters in seabird names like in sigma-é00
+        #cfile = open(fname, 'rb')
```

Comment:
I'll investigate this but we do need to read as binary and decode to make it compatible with py2 and py3.
Maybe we need another strategy there.